### PR TITLE
Ligra DMA

### PR DESCRIPTION
**Pull request by yodada**
In this PR:
 - use DMA in Ligra IO

### CODE REVIEW

File: examples/cuda/riscv.mk [hunk 1]
```diff
@@ -212,11 +212,10 @@ RISCV_ARCH_OP     := rv32imaf
 # CCPPFLAGS are common between GCC and G++
 RISCV_CCPPFLAGS += $(RISCV_OPT_LEVEL)
 RISCV_CCPPFLAGS += -march=$(RISCV_ARCH_OP)
-RISCV_CCPPFLAGS += -g
+RISCV_CCPPFLAGS += -fno-rtti
```
Comment (mrutt92) [inline examples/cuda/riscv.mk:215]: I thought i added this to appl-riscv.mk

**Review by mrutt92 — COMMENTED** [review]
I think there's some changes in here related to param-base-matmul. Did you want to make that it's own PR or do you not care?

**Comment by yodada** [comment]
> I think there's some changes in here related to param-base-matmul. Did you want to make that it's own PR or do you not care?

Oh I just wanted to merge the changes to bfs on top of #5

File: examples/cuda/riscv.mk (hunk 1 above)
Comment (yodada) [reply examples/cuda/riscv.mk:215]: ah right! I'll remove it

**Pull request closed without merging**